src/App/Link.h

```diff
@@ -208,6 +227,13 @@ class AppExport LinkBaseExtension : public App::DocumentObjectExtension
     typedef std::map<std::string, PropInfo> PropInfoMap;
     virtual const PropInfoMap &getPropertyInfoMap() const;
 
+    enum LinkCopyOnChangeType {
```

Comment:
Can this be a made a scoped enum instead?

Reply:
I think I'll leave this for now, or do you know any easier way to interoperate with `PropertyEnumeration`?